docs/auth.md

```diff
@@ -303,6 +303,15 @@ var sendVerificationEmail = function() {
 };
 ```
 
+### Property - isEmailVerified
+
+This property returns a boolean representing whether the user has verified their email or not by following the link sent by `sendVerificationEmail()`
+
```

Comment:
Please add a note here so users can know they _must_ re authenticate the user to get the latest values. Otherwise they will keep getting false.